src/components/v2v3/V2V3Project/ProjectSafeDashboard/juiceboxTransactions/reconfigureFundingCyclesOf/ReconfigurationRichPreview.tsx

```diff
@@ -28,10 +28,18 @@ export function ReconfigureRichPreview({
   } = useContext(ThemeContext)
 
   if (!contracts) return null
-
-  const dataResult: unknown = JBController?.interface?.parseTransaction({
-    data: transaction.data ?? '',
-  }).args
+  let dataResult: unknown
+  try {
+    dataResult = JBController?.interface?.parseTransaction({
+      data: transaction.data ?? '',
+    }).args
+  } catch (e) {
+    return (
+      <div style={{ margin: '1rem 3rem 0' }}>
```

Comment:
Noticed a weird error on some of the latest JB transactions. 

<img width="836" alt="Screen Shot 2022-10-04 at 10 04 46 pm" src="https://user-images.githubusercontent.com/96150256/193818699-acae7bc1-a2bc-4b17-8ab8-1a999d3f28ec.png">

Adding this `catch` for now